CSVLintNppPlugin/PluginInfrastructure/Lexer.cs

```diff
@@ -837,5 +859,34 @@ public static IntPtr GetSubStyleBases(IntPtr instance)
             // Fill styles with a byte for each style that can be split into substyles.
             return IntPtr.Zero;
         }
+
+        // --- iLexer5 Member Function Definitions ---
+
+        // virtual const char * SCI_METHOD GetName() = 0
+        public static IntPtr GetName(IntPtr instance)
+        {
+            /*  returns Lexer Language.
+             *  Equivalent to editor.getLexerLanguage() in PythonScript.
+             */
+            return Marshal.StringToHGlobalAnsi(ILexer.Name);
+        }
+
+        // virtual int SCI_METHOD GetIdentifier() = 0
+        public static int GetIdentifier(IntPtr instance)
+        {
+            /*  returns Lexer ID.
+             *  Equivalent to editor.getLexer() in PythonScript.
+             */
+            return LexerID;
+        }
+
+        // virtual const char * SCI_METHOD PropertyGet() = 0
+        public static IntPtr PropertyGet(IntPtr instance, IntPtr key)
+        {
+            /*  returns Property Value.
+             *  Equivalent to editor.getProperty('key') in PythonScript.
+             */
+            return Marshal.StringToHGlobalAnsi("N/A");
```

Comment:
There are a bunch of `editor.SetProperty()` calls in [Main.cs](https://github.com/BdR76/CSVLint/blob/master/CSVLintNppPlugin/Main.cs). While implementing this method, I tried to at least return these property values that were set earlier. But the `Main.editor.GetProperty()` (or internally, the `SCI_GETPROPERTY`) call was surprisingly returning empty even for those key values set earlier within the plugin. This very likely seems to be a regression with the Lexilla5 code rather than Notepad++. When this issue gets fixed in Lexilla, this line will need to be changed to:
```C#
return Marshal.StringToHGlobalAnsi(Main.editor.GetProperty(Marshal.PtrToStringAnsi(key)));
```
Also, for this line to compile, the `Main.editor` will need its access modified to be `public`.

@rdipardo, you write:
>The method signature has `IntPtr` as the return type, so just return `IntPtr.Zero` like `ILexer.DescribeWordListSets` above.

But there are other methods in `Lexer.cs` that also have the `IntPtr` return type and are actually returning `return Marshal.StringToHGlobalAnsi(...)`. Here is the list:
```C#
public static IntPtr PropertyNames(...);
public static IntPtr DescribeProperty(...);
public static IntPtr NameOfStyle(...);
public static IntPtr TagsOfStyle(...);
public static IntPtr DescriptionOfStyle(...);
public static IntPtr GetName(...);
```

However, if @BdR76  decides that we should return an empty string in `PropertyGet` until Lexilla fixes their issue, then of course, we can change the return line to `return IntPtr.Zero;`.

Reply:
Here's how Python and Lua interpret `IntPtr.Zero`:

![nullptr-for-lex-props](https://user-images.githubusercontent.com/59004801/171238288-5b1d2461-cb8c-4f1d-8e90-e7513c41a9d6.png)

Reply:
Thank you @rdipardo. I have just now pushed a commit with change.